src/server/mod.rs

```diff
@@ -51,13 +53,14 @@ where
             let app = App::new()
                 .app_data(Data::new(runtime_context.clone()))
                 .wrap(Logger::default())
-                .route("/", web::get().to(index))
+                .route("/", web::get().to(index::<Context, KeyManager>))
+                .route("/status", web::get().to(index::<Context, KeyManager>))
```

Comment:
Shouldn't this call `index` from status? Or am I missing something?

Reply:
mb again